src/app/tables/table.component.html

```diff
@@ -83,6 +85,11 @@ <h2 class="table-title" *ngIf="title">
                   </ng-template>
                 </td>
               </ng-container>
+              <td *ngIf="nodeStoryUrls[item[uidColumn]] !== undefined">
```

Comment:
First part of the suggestion:
```suggestion
              <td>
```